inginious/frontend/plugins/manual_scoring/static/js/code_field.js

```diff
@@ -1,52 +0,0 @@
-const LANGUAGES = {
```

Comment:
Why did you remove this? Check again after these last changes that everything is working ok.

Reply:
A few weeks ago I had problem minifying files and the problem was related to the content of that file. The file had worked normally (without min files) and I hadn't modified a lot, therefore I deleted, modify the file, the name, etc. To know what was the matter. The problem was how I declared a variable, but with the updates with master, two files stayed. The content is the same, just changed the name of the class.

Reply:
So there is no problem right? can I merge this without introducing a bug?

Reply:
Yes